src/api-reference/invoice/purchase-request.markdown

```diff
@@ -41,7 +40,7 @@ https://emea.api.concursolutions.com/purchaserequest/v1/
 
 Once you have registered your application, read about the [Purchase Request API endpoints](/api-reference/invoice/purchase-request-endpoints.html), or click the button to download a request collection for Postman.
```

Comment:
There should be no links to Postman, cUrl or any other tooling.

Reply:
@charlieconcur 
Per your earlier comment on my PR#1477 I added links to post man, do you recommend to remove them now? I see receipt V4 API that has post man links. I do see postman files being valuable for our customers to quickly connect to our API's for verification purpose. 
Please let me know.

Reply:
Please remove them. I'm working with the Receipts documentation now to remove them there as well.

Reply:
I have removed references to Postman in 7df33a5. Please check

Reply:
@charlieconcur Please let us know if this looks good.

Reply:
@charlieconcur This is good to move to prod, please review and move forward with this.